definitions/contested/json/ComplexTypes.json

```diff
@@ -184,7 +184,7 @@
     "ID": "FR_ct_genaralOrder",
     "ListElementCode": "additionalDocument",
```

Comment:
is this being used?

Reply:
would still be used to show the full collection in the judicary tab, this will mean the document is shown with the label document